pkgs/by-name/as/asdbctl/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  udev,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "asdbctl";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "juliuszint";
+    repo = "asdbctl";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-S5m1iQlchGKc0PODQNDHpNzaNXRepmk5zfK5aXdiMiM=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-OPmnGh6xN6XeREeIgyYB2aeHUpdQ5hFS5MivcTeY29E=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
+    udev
+  ];
+
+  postInstall = ''
+    install -Dm644 \
```

Comment:
Does this need to have write permissions? Could it just be `444`?

Reply:
Fixed.